File: library/BackEnd/routes/UserRoute.py
"""
User Route
Handle routing for User realted operations
"""


from fastapi import APIRouter
from models.UserModel import load_Users

router = APIRouter()



"""
Fetch the User data from JSON file

Returns:
    dict:User data

"""
@router.get("/Users")
def get_Users():
    return load_Users()

File: library/BackEndNode/routes/UserRoute.js

const { readFile } = require("../models/FileModel")

const express = require("express");
const router = express.Router();


/**
 * GET /Users
 * get data of user.
 * @route GET /Users
 * @returns {Object} List of all users 
 */

router.get("/Users", async (req, res) => {
    res.json(await readFile('data/Users.json'));
})


module.exports = router;

